packages/relay/index.js

```diff
@@ -1,8 +1,10 @@
 // @flow
 
 import * as React from 'react';
-import Relay from '@kiwicom/relay';
-import { type GraphQLTaggedNode as _GraphQLTaggedNode } from 'react-relay'; // eslint-disable-line
+import {
+  commitMutation as _commitMutation,
```

Comment:
Is it just to put emphasis on its _internal API_ nature?

Reply:
No, it is because I do export a const named commitMutation, so there would be a naming collision

Reply:
Ah right. 😄 I should have expanded the file view!